regimes/it/categories.go

```diff
@@ -90,19 +92,24 @@ var categories = []*tax.Category{
 		},
 	},
 	{
-		Code:     TaxCategoryRA,
+		// IT: https://www.agenziaentrate.gov.it/portale/imposta-sul-reddito-delle-persone-fisiche-irpef-/aliquote-e-calcolo-dell-irpef
+		// EN: https://www.agenziaentrate.gov.it/portale/web/english/information-for-specific-categories-of-workers
+		Code:     TaxCategoryIRPEF,
 		Retained: true,
 		Name: i18n.String{
-			i18n.EN: "RA",
-			i18n.IT: "RA",
+			i18n.EN: "IRPEF",
+			i18n.IT: "IRPEF",
 		},
 		Desc: i18n.String{
-			i18n.EN: "Withholding Tax",
-			i18n.IT: "Ritenuta d'Acconto",
+			i18n.EN: "Personal Income Tax",
+			i18n.IT: "Imposta sul Reddito delle Persone Fisiche",
+		},
+		Meta: cbc.Meta{
+			KeyFatturaPATipoRitenuta: "RT01",
```

Comment:
I would love to be able to fetch this from the provider through a clean interface. Could we apply some variation of the "summary" pattern here?